src/domain/pset.ts

```diff
@@ -13,14 +18,15 @@ import {
 } from '../application/account';
 import type { WalletRepository, AppRepository, TaxiRepository } from './repository';
 import type { CoinSelection } from './transaction';
+import { computeBalances } from './transaction';
 
-function estimateScriptSigSize(type: ScriptType): number {
+function estimateScriptSigSize(type: address.ScriptType): number {
   switch (type) {
-    case ScriptType.P2Pkh:
+    case address.ScriptType.P2Pkh:
       return 108;
-    case (ScriptType.P2Sh, ScriptType.P2Wsh):
+    case (address.ScriptType.P2Sh, address.ScriptType.P2Wsh):
```

Comment:
You have `address.ScriptType.P2Wsh` repeated and I didn't knew this syntax for the case clause 😄 

Maybe we should go for a more conservative approach?

```suggestion
    case address.ScriptType.P2Pkh:
      return 108;
    case address.ScriptType.P2Sh:
      return 35;
    case address.ScriptType.P2Wsh:
    case address.ScriptType.P2Tr:
    case address.ScriptType.P2Wpkh:
      return 1;
```

Reply:
Commited :heavy_check_mark: , just P2WSH must return 35